sing in.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+    
+  
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    
+</head>
+<body>
+    <div class="login_box">
+        <img src="icons8-user-64.png" alt="user">
+        <br>
+        <h1> Sign In</h1>
+        <form action="#">
+            <div class="input_box">
+           
+          
+       <input type="text" name="Email" placeholder="E-mail address">
+                <div class="icons"><i class="fas fa-envelope"></i></div>
+             </div>
+            
+            <div class="input_box">
+            
+           
+            <input type="password" name="password" placeholder="password">
+            <div class="icons">
+                <i class="fas fa-lock"> </i></div>
+              </div>
+            <div class="input_box">
+            
+            <input type="submit" name="submit" value="sign in">
+           </div>
+            <hr>
+            <br>
+            <div class="sign_up">
+                New here? <a href="#">Sign Up</a>
+            </div>
+        </form>
+    </div>
+    
+ </body>
+
+</html>
```

Comment:
### It is recommended to have a new line character at the end of the file
[Check why..](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)